presto-spi/src/main/java/com/facebook/presto/spi/session/SessionPropertyConfigurationManagerFactory.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.session;
+
+public interface SessionPropertyConfigurationManagerFactory
```

Comment:
I don't see any implementations for this interface.

Reply:
I only have these interfaces in SPI implemented in presto-facebook. I know it's strange. But they need to be here to enable anyone to add a plugin.

Reply:
See if you can write a test implementation of this as part of an integration test